src/overlays/actors/ovl_En_Karebaba/z_en_karebaba.h

```diff
@@ -1,16 +1,30 @@
 #ifndef _Z_EN_KAREBABA_H_
 #define _Z_EN_KAREBABA_H_
 
+/**
+ * File: z_en_karebaba.h
+ * Overlay: ovl_En_Karebaba
+ * Description: Withered Deku Baba enemies
+ */
```

Comment:
we dont have file headers like this in header files currently, you can remove this for the time being